doc: Update versions for 2.2.0 release (#877)
nrfconnect/sdk-nrfxlib: nrf_rpc/CHANGELOG.rst

> All notable changes to this project in the |NCS| are documented in this file.

Reviewer: ```suggestion
All the notable changes to this project are documented on this page.
```
